Added useRouter to /learn (#351)
zeit/next-site: www/pages/learn/basics/create-dynamic-pages/use-router.mdx

import Layout from '../../../../components/learn/Layout';

export const meta = {
  title: 'Create Dynamic Pages',
  courseId: 'basics',
  lessonId: 'create-dynamic-pages',
  stepId: 'use-router'
};

## useRouter

> As you can see, [useRouter](https://nextjs.org/docs#use-router) allows you to access the `router` object inside the page, it's a [React Hook](https://reactjs.org/docs/hooks-intro.html) and it can work in any component.

Reviewer: ```suggestion
As you can see, [useRouter](https://nextjs.org/docs#use-router) allows you to access the `router` object inside the page, it's a [React Hook](https://reactjs.org/docs/hooks-intro.html) and it can work with function components.
```

> As you can see, [useRouter](https://nextjs.org/docs#use-router) and [withRouter](http://localhost:3000/docs#using-a-higher-order-component) do the same, the difference remains in that `useRouter` is the recommended solution while working with [React Hooks](https://reactjs.org/docs/hooks-intro.html) and `withRouter` is a [High Order Component](https://reactjs.org/docs/higher-order-components.html). Both add the `router` object to a component and both work in any component.

Reviewer 2: ```suggestion
As you can see, [useRouter](https://nextjs.org/docs#use-router) and [withRouter](https://nextjs.org/docs#using-a-higher-order-component) do the same, the difference remains in that `useRouter` is the recommended solution while working with [React Hooks](https://reactjs.org/docs/hooks-intro.html) and `withRouter` is a [High Order Component](https://reactjs.org/docs/higher-order-components.html). Both add the `router` object to a component and both work in any component.
```
